### ISSUE TITLE
Feature comparison?

### ISSUE THREAD

**Issue by fkromer** [jaydenwindle/graphene-subscriptions#3]
The plug and play feel of the package is great. Would be helpful for potential users to have a feature comparison w.r.t. other integration packages like e.g. [DjangoChannelsGraphqlWs](https://github.com/datadvance/DjangoChannelsGraphqlWs). Are you planning to extend the functionality of the package?

**Comment by jaydenwindle** [comment]
Good idea! I'll take a look at the other packages and see if I can come up with some comparisons.

I'm definitely going to continue to develop this package. Is there any extra functionality in particular that you would like to see added?

**Comment by fkromer** [comment]
I like `DjangoChannelsGraphqlWs`s feature to beeing able to define callbacks `on_connect()`, `subscribe()` and `publish()`. Especially for `on_connect()` there is no equivalent in `graphene-subscriptions` I guess. It seems like `graphene-subscriptions` is tightly coupled to Django models. Would be nice to allow a way to provide data via graphene fields which are not linked to a model.

**Comment by jaydenwindle** [comment]
That's great feedback, thanks! Callbacks would definitely be useful. I'll work on adding them.

The `graphene-subscriptions` docs definitely center on using Django models right now, but you can actually use any Graphene type in your subscriptions. There's some documentation about it [here](https://github.com/jaydenwindle/graphene-subscriptions#custom-events), but it could definitely use some extra explanation :)

**Comment by fkromer** [comment]
> That's great feedback, thanks! Callbacks would definitely be useful. I'll work on adding them.

Cool!

> The `graphene-subscriptions` docs definitely center on using Django models right now, but you can actually use any Graphene type in your subscriptions. There's some documentation about it [here](https://github.com/jaydenwindle/graphene-subscriptions#custom-events), but it could definitely use some extra explanation :)

Ah, right. That's possible of course.
